roles/kubernetes-apps/external_cloud_controller/openstack/templates/external-openstack-cloud-controller-manager-role-bindings.yml.j2

```diff
@@ -1,40 +0,0 @@
-apiVersion: v1
```

Comment:
This approach makes us loose the history of the file changes, is there a good reason to do this instead of modifying the file contents in-place?

Reply:
There's no particular reason other than that since it's only one object right now I thought it was unnecessary. But I can change back